tests/FakeItEasy.Tests/Configuration/RuleBuilderTests.cs

```diff
@@ -78,7 +78,7 @@ public void Returns_with_call_function_should_return_delegate()
 
             var returned = config.ReturnsLazily(x => x.Arguments.Get<int>(0));
 
-            returned.Should().Be(config.ParentConfiguration);
+            returned.Should().BeSameAs(config);

```

Comment:
Yes, probably... but I'd rather drop this test anyway, for the same reason as above (GTT)